### PR TITLE
LaTeX: fix usage of PAPER envvar, which got broken at Sphinx 1.5

### PR DESCRIPTION
**Pull request by jfbu**
Fix: #5234

@tk0miya I can not test the Windows make.bat, I hope my commit is ok for it.

### CODE REVIEW

File: sphinx/make_mode.py [hunk 1]
```diff
@@ -143,7 +143,7 @@ def run_generic_build(self, builder, doctreedir=None):
         papersize = os.getenv('PAPER', '')
         opts = self.opts
         if papersize in ('a4', 'letter'):
-            opts.extend(['-D', 'latex_elements.papersize=' + papersize])
+            opts.extend(['-D', 'latex_elements.papersize=' + papersize + 'letter'])
```
Comment (tk0miya) [inline sphinx/make_mode.py]: `'paper'`?

**Review by tk0miya — COMMENTED** [review]
LGTM with nits

File: sphinx/make_mode.py (hunk 1 above)
Comment (jfbu) [reply sphinx/make_mode.py]: OMG! yes of course sorry!

**Comment by jfbu** [comment]
@tk0miya updated :-)

**Comment by tk0miya** [comment]
confirmed. Go ahead!

**Comment by jfbu** [comment]
Thanks for quick review @tk0miya, merged!

**Pull request merged**